docs/working-with-projects/working-with-images/preheat-images.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Preheat Images
+weight: 30
+---
+Preheat action is policy-based driven way, and the scope of preheat policy is limited in the project level. Preheat 
```

Comment:
preheat action is policy driven, and is scoped to the project within which it's created